dev/release/source_build.sh

```diff
@@ -45,6 +45,8 @@ main() {
     mv "${base_name}/" "${base_name}.tmp/"
     cp -R -d "${base_name}.tmp" "${base_name}"
     rm -rf "${base_name}.tmp/"
+    MESON_SOURCE_ROOT="${base_name}/python" MESON_DIST_ROOT="${base_name}/python" \
```

Comment:
My bash knowledge is really bad, so happy to be shown a better way of specifying these variables to be passed to the command :-)